js/jquery.terminal-1.5.0.js

```diff
@@ -4858,7 +4872,12 @@
                 if (options.escape) {
                     safe = safe.replace(/\\(["'() ])/g, '\\?$1');
                 }
-                var regex = new RegExp('^' + safe);
+                var regex;
+                if (options.caseSensitive) {
+                    regex = new RegExp('^' + safe);
+                } else {
+                    regex = new RegExp('^' + safe, 'i');
```

Comment:
it's shorter to use `var regex = new RegExp('^' + safe, options.caseSensitive ? '' : 'i');`